bottles/frontend/views/bottle_details.py

```diff
@@ -472,18 +472,19 @@ def __confirm_delete(self, widget):
         to the bottles list.
         """
         def handle_response(_widget, response_id):
-            if response_id == "ok":
+            if response_id == "delete":
                 RunAsync(self.manager.delete_bottle, config=self.config)
                 self.window.page_list.disable_bottle(self.config)
             _widget.destroy()
 
         dialog = Adw.MessageDialog.new(
             self.window,
-            _("Confirm"),
-            _("Are you sure you want to delete this Bottle and all files?")
+            _(f"Are you sure you want to permanently delete \"{self.config['Name']}\"?"),
```

Comment:
this will be an issue in translations, use `"Are you sure you want to permanently delete {}?".format({self.config['Name'])` instead